tests/pytests/unit/modules/test_aptpkg.py

```diff
@@ -1409,3 +1509,551 @@ def test_sourceslist_architectures(repo_line):
                     assert source.architectures == ["amd64", "armel"]
                 else:
                     assert source.architectures == ["amd64"]
+
+
+@pytest.mark.parametrize(
+    "pkg,arch",
+    [
+        ("zsh", "amd64"),
+        ("php", "x8664"),
```

Comment:
```suggestion
        ("php", "x86_64"),
```
Was this string originally correct? Or it my suggestion the correct one?